modules/apps/portal-vulcan/portal-vulcan-impl/src/main/java/com/liferay/portal/vulcan/internal/jaxrs/writer/interceptor/EntityExtensionWriterInterceptor.java

```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.portal.vulcan.internal.jaxrs.writer.interceptor;
+
+import com.liferay.portal.vulcan.internal.jaxrs.extension.ExtendedEntity;
+import com.liferay.portal.vulcan.jaxrs.context.ExtensionContext;
+
+import java.io.IOException;
+
+import java.util.Optional;
+
+import javax.ws.rs.core.Context;
+import javax.ws.rs.ext.Provider;
+import javax.ws.rs.ext.Providers;
+import javax.ws.rs.ext.WriterInterceptor;
+import javax.ws.rs.ext.WriterInterceptorContext;
+
+/**
+ * @author Javier de Arcos
+ */
+@Provider
+public class EntityExtensionWriterInterceptor implements WriterInterceptor {
+
+	@Override
+	public void aroundWriteTo(WriterInterceptorContext writerInterceptorContext)
+		throws IOException {
+
+		Optional.ofNullable(
+			_providers.getContextResolver(
+				ExtensionContext.class, writerInterceptorContext.getMediaType())
+		).map(
+			contextResolver -> contextResolver.getContext(
+				writerInterceptorContext.getType())
+		).ifPresent(
+			extensionContext -> _extendEntity(
+				writerInterceptorContext, extensionContext)
+		);
+
+		writerInterceptorContext.proceed();
+	}
+
+	private void _extendEntity(
+		WriterInterceptorContext writerInterceptorContext,
```

Comment:
Sort :(

Reply:
I always forgot about this. I don't know if it is a good thing or a bad thing ...😉 

Thank you for the catch!